docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/customizing-policy.adoc

```diff
@@ -0,0 +1,30 @@
+= Customizing Policy
+
+An Enterprise Contract Policy defines which checks are performed by EC during integration tests and
+release. If EC reports violations, the best course of action, in most cases, is to adjust the build
+process to mitigate the violation. For example, if EC reports that a certain required scanner was
+not executed, add the scanner to your build pipeline.
+
+However, there are times where mitigating the violation is not an option. In this case, customizing
+the Policy is the only option.
+
+First, xref:/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc[identify]
+which Policy is being used to verify the Components in your Application.
+
+Second, determine if this is a Policy you can modify. Policies are often shared between different
+users. Adding an exception to a shared Policy may not be in the best interest of all users. Creating
+a new Policy for your particular use case may be a better approach. You can copy an existing Policy
+as a starting point.
+
+Third, update your Policy to exclude a certain violation. See the
+link:https://enterprisecontract.dev/docs/ec-cli/main/configuration.html[Policy Configuration] docs
+for details and examples.
+
+Fourth, update link:https://konflux-ci.dev/docs/how-tos/testing/integration/[integration tests] and
```

Comment:
`update your` to match style of the third point above.

Reply:
done. ty!